app/controllers/users/invitations_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Users::InvitationsController < Devise::InvitationsController
   before_action :delete_user_record, if: :user_should_be_cleared?, only: :create
   before_action :return_user_to_invite_page, if: :user_is_invalid?, only: :create
-  before_action :add_organisation_to_params, only: :create
+  before_action :add_organisation_to_params, unless: :super_admin?, only: :create
 
 private
 
```

Comment:
can `authenticate_inviter!` (below this line) be explicit about allowing super admins please?
being a super_admin doesn't automatically give the permission to manage a team, which will lead to some unusual potentially hard to diagnose issues down the line

Reply:
Thinking about this further, we've been talking about read-only super admins.
I'll leave it as is for now, but may consider adding a test to make it explicit a super admin without the team management perm can't invite team members.